scripts/build/config.js

```diff
@@ -66,6 +67,14 @@ const parsers = [
   {
     input: "src/language-html/parser-parse5.js",
     target: "node"
+  },
+  {
+    input: "src/language-yaml/parser-yaml.js",
+    target: "universal",
+    bundler: "webpack",
```

Comment:
Rollup didn't work?

Reply:
```
TypeError: lines_and_columns_1.default is not a constructor
```

Reply:
Possibly related to this: https://github.com/rollup/rollup-plugin-commonjs/issues/160

`line-and-colum` is shipped with both CJS and ESM versions:

```js
// ESM
export default LineAndColumns;

// CJS
exports["default"] = LineAndColumns;
```
The distributed `yaml-unist-parser` version has a `require("line-and-columns")["default"]`. But Rollup uses the ESM version by default and by its semantics, when using ESM `require("line-and-columns") === LineAndColumns`.

I aliased `line-and-columns` in rollup bundle to point to the CJS version.